cloudify_aws/ec2/resources/ebs.py

```diff
@@ -232,8 +233,30 @@ def create(ctx, iface, resource_config, **_):
     params = utils.clean_params(
         dict() if not resource_config else resource_config.copy())
 
-    # Create ebs resource
-    create_response = iface.create(params)
+    # Actually create ebs resource
+    region_name = ctx.node.properties['client_config']['region_name']
+    use_available_zones = params.pop('UseAvailableZones', False)
```

Comment:
Should be `ctx.node.properties['use_available_zones']`, or is that a real parameter for create?

Reply:
@EarthmanT  we added that to the properties , and if passed to the API it will fail , so I just pop it to remove it from being passed to the API